pkg/ui/src/views/cluster/containers/events/index.tsx

```diff
@@ -53,7 +53,7 @@ export interface EventRowProps {
 
 export function getEventInfo(e: Event$Properties): SimplifiedEvent {
   return {
-    fromNowString: TimestampToMoment(e.timestamp).fromNow()
+    fromNowString: TimestampToMoment(e.timestamp).format("MMM DD, YYYY [at] h:mm A")
```

Comment:
Can we put the timestamp format string somewhere shared? Is there a utils file somewhere or a constants file?